singlestoredb/management/utils.py

```diff
@@ -152,6 +152,26 @@ def get_organization() -> Optional[str]:
     return None
 
 
+def get_cluster_id() -> Optional[str]:
+    """Return the cluster id for the current token or environment."""
+    return os.environ.get('SINGLESTOREDB_CLUSTER')
```

Comment:
You might add `or None` to the end of the returns in these several functions. That way if the env var is an empty string, it will still return `None` and not a string.